src/lang/en.json

```
@@ -753,6 +753,11 @@
     "smseagleUrl": "Your SMSEagle device URL",
     "smseagleEncoding": "Send as Unicode",
     "smseaglePriority": "Message priority (0-9, default = 0)",
+    "smspartnerApikey": "API Key",
+    "smspartnerApiurl": "Find your API key in your dashboard: ",
+    "smspartnerPhoneNumber": "Phone number(s) (multiple numbers, separate by `,`)",
+    "smspartnerSenderName": "SMS Sender Name",
+    "smspartnerSenderNameInfo": "Must be between 3 and 11 inclusive and must not contain any special characters.",
```

what do you think about this alternative?

```suggestion
    "smspartnerSenderNameInfo": "Must be between 3..=11 regular characters",
```